client/my-sites/earn/memberships/products.jsx

```diff
@@ -177,152 +143,28 @@ class MembershipsProductsSection extends Component {
 		}
 		this.setState( { deletedProductId: null } );
 	};
-	handleCurrencyChange = ( event ) => {
-		const { value: currency } = event.currentTarget;
-
-		this.setState( ( state ) => ( {
-			editedPrice: { ...state.editedPrice, currency },
-		} ) );
-	};
-	handlePriceChange = ( event ) => {
-		const value = parseFloat( event.currentTarget.value );
-
-		this.setState( ( state ) => ( {
-			editedPrice: { ...state.editedPrice, value },
-		} ) );
-	};
-	handlePayWhatYouWant = ( newValue ) => this.setState( { editedPayWhatYouWant: newValue } );
-	handleMultiplePerUser = ( newValue ) => this.setState( { editedMultiplePerUser: newValue } );
 
-	onNameChange = ( event ) => this.setState( { editedProductName: event.target.value } );
-	onSelectSchedule = ( event ) => this.setState( { editedSchedule: event.target.value } );
-	isFormValid = ( field ) => {
-		if (
-			( field === 'price' || ! field ) &&
-			! isValidCurrencyAmount( this.state.editedPrice.currency, this.state.editedPrice.value )
-		) {
-			return false;
-		}
-		if ( ( field === 'name' || ! field ) && this.state.editedProductName.length === 0 ) {
-			return false;
-		}
-		return true;
-	};
+	renderAddNewPlanDialog() {
+		return (
+			<RecurringPaymentsPlanAddEditModal
+				isVisible={ this.state.showDialog === 'addNewPlan' }
+				currencyList={ currencyList }
+				minimumCurrencyTransactionAmount={ minimumCurrencyTransactionAmount }
+				onClose={ this.onCloseDialog }
+				productId={ null }
```

Comment:
`productId` doesn't appear to be a prop of `RecurringPaymentsPlanAddEditModal`, maybe

```suggestion
				product={ null }
```

Reply:
Well spotted, yes – fixed.